Java/Java.sublime-syntax

```diff
@@ -253,25 +373,33 @@ contexts:
       scope: keyword.operator.arithmetic.java
     - match: (!|&&|\|\|)
       scope: keyword.operator.logical.java
-    - match: (?<=\S)\.(?=\S)
-      scope: keyword.operator.dereference.java
+    - match: \.
+      scope: punctuation.accessor.java
```

Comment:
PHP uses `\` for namespace "accessor", in the way that C++ uses `::`, although right now I believe in PHP it is scoped as a `punctuation.separator.namespace`.